code/modules/elevator/elevator_indicator.dm

```diff
@@ -0,0 +1,35 @@
+/obj/machinery/elevator_indicator
+	name = "floor indicator"
+	icon = 'icons/obj/elevator.dmi'
```

Comment:
this needs a desc, and it should probably also list the text on it when examined

Reply:
control panel also needs a desc